Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "nitro-cli"
-version = "1.0.8"
+version = "1.1.1"
```

Comment:
why 1.1.1 and not 1.1.0? Also, why is this a version change, I don't see anything in the changelog that would indicate any significant enough functionality change.

Reply:
Right, revision could be 0. The version change was more of a convention to mark that versions before 1.1 (namely 1.0-5 which is currently available to customers on AL2 instances) misbehave on `yum update`s. Versions from 1.1 fix this update problem.